pom.xml

```diff
@@ -20,7 +20,7 @@
 
   <groupId>com.google.cloud.datafusion</groupId>
   <artifactId>ariba-plugins</artifactId>
-  <version>0.1.0-SNAPSHOT</version>
+  <version>0.2.0-SNAPSHOT</version>
```

Comment:
why did you bump the version?

Reply:
That was for tracking purpose, moved it back to 1.0